includes/malloc.h

```diff
@@ -0,0 +1,31 @@
+#ifndef MALLOC_H
+#define MALLOC_H
+
+#include <stdint.h>
+#include <unistd.h>
+
+extern uint32_t _ebss;
+
+#define HEAP_START ((void*)&_ebss);
+
+#define ALIGN(x) (((((x) - 1) >> 2) << 2) + 4)
```

Comment:
What is this aligning?

Reply:
Aligning by 4 bytes. As far as I know, it's faster than using "%"